gcloud-java-examples/README.md

```diff
@@ -87,6 +87,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
+
+     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
+     Compile the JAR with:
+    ```
+    mvn package -DskipTests
+    ```
+    Then run the sample with:
+    ```
+    java -cp gcloud-java-contrib/gcloud-java-nio/target/gcloud-java-nio-0.1.4-SNAPSHOT-jar-with-dependencies.jar:gcloud-java-examples/target/gcloud-java-examples-0.1.4-SNAPSHOT.jar com.google.gcloud.examples.Stat --check
```

Comment:
It turns out I had to fix the shaded jar configuration, because by default the shade plugin does not shade the jar (I was surprised to discover). The option we need is called "relocation" and it is *not* described in the "[usage](https://maven.apache.org/plugins/maven-shade-plugin/usage.html)" page.